docs/faq.md

```diff
@@ -44,11 +44,9 @@ a secret named `argocd-initial-admin-secret`.
 
 To change the password, edit the `argocd-secret` secret and update the `admin.password` field with a new bcrypt hash.
 
-!!! note "Generating a bcrypt hash"
-    Use a trustworthy, offline `bcrypt` implementation such as the [Python bcrypt library](https://pypi.org/project/bcrypt/) to generate the hash.
+  Use the following command to generate a bcrypt hash for `admin.password`
```

Comment:
```suggestion
!!! note "Generating a bcrypt hash"
    Use the following command to generate a bcrypt hash for `admin.password`
```

I suggest we leave this in a nice, loud box. You can preview it here: https://argo-cd--10934.org.readthedocs.build/en/10934/faq/#i-forgot-the-admin-password-how-do-i-reset-it